first_hour_on_vwb/creating_a_data_collection.ipynb

```diff
@@ -0,0 +1,709 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "class AddReleaseNotesWidget(object):\n",
+    "    def __init__(self, prev_widget):\n",
+    "        self.label = widgets.Label(value = 'Please provide appropriate values in the input boxes.')\n",
+    "        self.data_collection_ids = self.get_data_collections()\n",
+    "        if prev_widget and prev_widget.get_workspace_id() != '':\n",
+    "            self.input_workspace_id = wu.DropdownInputWidget(self.data_collection_ids,prev_widget.get_workspace_id(),\"Data Collection:\").get()\n",
+    "        else:\n",
+    "            self.input_workspace_id = wu.DropdownInputWidget(self.data_collection_ids,self.data_collection_ids[0],\"Data Collection:\").get()\n",
+    "        self.input_version = wu.TextInputWidget(\"<VERSION>\",\"Version:\").get()\n",
+    "        self.input_notes_url = wu.TextInputWidget(\"<NOTES_URL>\",\"Notes URL:\").get()        \n",
+    "        self.button = wu.StyledButton('Attach notes','Click to attach release notes','check').get()\n",
+    "        self.output = widgets.Output()\n",
+    "        self.initial_fields = [\n",
+    "            self.label,\n",
+    "            self.input_workspace_id,\n",
+    "            self.input_version,\n",
+    "            self.input_notes_url,\n",
+    "            self.button,\n",
+    "            self.output\n",
+    "        ]\n",
+    "        self.vb = widgets.VBox(children=self.initial_fields, layout=wu.vbox_layout)\n",
+    "        self.button.on_click(self.add_release_notes)\n",
+    "\n",
+    "    def get_data_collections(self):\n",
+    "        result = subprocess.run([\"terra\",\"workspace\",\"list\",\"--format=JSON\"],capture_output=True,text=True)\n",
+    "        ids_list = wu.list_data_collections(result.stdout)\n",
+    "        # Insert empty string to display as value of dropdown until changed by user.\n",
+    "        ids_list.insert(0,'')\n",
+    "        return ids_list\n",
+    "\n",
+    "    def add_release_notes(self,b):\n",
+    "        with self.output:\n",
+    "            # Save values as variables for reuse.\n",
+    "            workspace_id = self.input_workspace_id.value.strip()\n",
+    "            version = self.input_version.value.strip()\n",
+    "            notes_url = self.input_notes_url.value.strip()\n",
+    "            \n",
+    "            # 1) Point cloud env to target data-collection workspace.\n",
+    "            setWorkspaceCommand = f\"terra workspace set --id={workspace_id}\"\n",
+    "            setWorkspaceResult = subprocess.run(setWorkspaceCommand, shell = True, capture_output = True, text = True)\n",
+    "            print(setWorkspaceResult.stderr) if not setWorkspaceResult.stdout else print(setWorkspaceResult.stdout)\n",
+    "\n",
+    "            # 2) Get file tree from Terra CLI.\n",
+    "            tree_dictionary = vfu.parse_tree()\n",
```

Comment:
At this point I got an error `AttributeError: module 'vwb_folder_utils' has no attribute 'parse_tree'`